maintaining.rst

```diff
@@ -0,0 +1,195 @@
+***********************
+Maintaining collections
+***********************
+
+.. contents:: Topics
+
+What is the collection maintenance
+==================================
+
+A collection is an Open Source project.
+
+If a community of any Open Source project can be logically divided into three groups: core members, contributors, and users, the collection maintainers are the core members of the collection.
```

Comment:
Let's not use the term `core` to avoid confusion with the Core Team (who look after ansible-core).

Reply:
Good spot

Reply:
Changed, thanks